package-overrides/github/Eonasdan/bootstrap-datetimepicker@4.7.14.json

```diff
@@ -0,0 +1,27 @@
+{
+  "format": "global",
+  "main": "build/js/bootstrap-datetimepicker.min.js",
+  "registry": "github",
+  "dependencies": {
+    "moment": "npm:moment",
+    "jquery": "github:components/jquery",
+    "bootstrap": "github:twbs/bootstrap@^3.0.0",
+    "css": "github:systemjs/plugin-css"
```

Comment:
If you did want to simplify the above, you can just write `css: '*', bootstrap: '^3.0.0', 'moment': '*', jquery: '*'` which is equalvent to the above.